### PR TITLE
Cavity service skeleton

### PR DESCRIPTION
**Pull request by youssefM1999**
I trained and tested a model to identify cavities on teeth. The accuracy is 92.8% on the test samples.

### CODE REVIEW

File: cavity-service/cavity_service/CavityModel.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import matplotlib.pyplot as plt
+from sklearn.linear_model import LogisticRegression
+from sklearn import preprocessing
+from PIL import Image
+from sklearn.metrics import confusion_matrix, accuracy_score
+from pathlib import Path
+
+# current_dir = Path(os.getcwd())
+# current_str = str(current_dir)
+# print("Current: " + current_str)
```
Comment (ahmedmsabry2000) [inline cavity-service/cavity_service/CavityModel.py:13]: delete commented code

File: cavity-service/cavity_service/CavityModel.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+folder_training_0 = "data/teeth_dataset/training/without_caries/"
+folder_training_1 = "data/teeth_dataset/training/caries/"
+folder_testing_0 = "data/teeth_dataset/testing/without_caries/" 
+folder_testing_1 = "data/teeth_dataset/testing/caries/"
+
+# ../data/teeth_dataset/training/
+# folder_training_cav = '../data/teeth_dataset/training/caries'
+# folder_training_no_cav = '../data/teeth_dataset/training/without_caries'
+# folder_testing_cav = '../data/teeth_dataset/testing/caries'
+# folder_testing_no_cav = '../data/teeth_dataset/testing/without_caries'
```
Comment (ahmedmsabry2000) [inline cavity-service/cavity_service/CavityModel.py:24]: delete commented code

**Comment by ahmedmsabry2000** [comment]
remove jpg files from commit
